graphql/version_resolver.go

```diff
@@ -160,6 +161,31 @@ func (r *versionResolver) ExternalLinksForMetadata(ctx context.Context, obj *res
 	return externalLinks, nil
 }
 
+// GeneratedTaskCounts is the resolver for the generatedTaskCounts field.
+func (r *versionResolver) GeneratedTaskCounts(ctx context.Context, obj *restModel.APIVersion) (map[string]interface{}, error) {
+	versionID := utility.FromStringPtr(obj.Id)
+	v, err := model.VersionFindOneId(*obj.Id)
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error while finding version with id: `%s`: %s", versionID, err.Error()))
+	}
+	if v == nil {
+		return nil, ResourceNotFound.Send(ctx, fmt.Sprintf("Version does not exist: `%s`", versionID))
+	}
+
+	res := map[string]interface{}{}
+	versionGeneratorTasks, err := task.Find(bson.M{
+		task.VersionKey:      versionID,
+		task.GenerateTaskKey: true,
+	})
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error while finding generator tasks from version: `%s`: %s", versionID, err.Error()))
+	}
```

Comment:
nit. formatting and regular quotes
```suggestion
	if err != nil {
		return nil, InternalServerError.Send(ctx, fmt.Sprintf("finding generator tasks from version '%s': %s", versionID, err.Error()))
	}
```